python/activity_stream/data_manager.py

```diff
@@ -722,37 +726,39 @@ def __db_insert_activity_updates(
                         INSERT OR REPLACE INTO activity(activity_id, payload, created_at)
                         SELECT ?, ?, datetime('now')
                     """
+                    params = (activity_id, blob)
```

Comment:
This was actually a bug in the current implementation! We passed too many bindings in.